all_tests.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
 
-SKIP_TESTS_THAT_REQUIRE_A_SEPARATE_PROCESS=1 python -m unittest
-STATUS_ALL_TESTS=$?
+#SKIP_TESTS_THAT_REQUIRE_A_SEPARATE_PROCESS=1 python -m unittest
+#STATUS_ALL_TESTS=$?
 
 python -m unittest sample_factory_examples.tests.test_example
 STATUS_TEST_EXAMPLE=$?
 
-python -m unittest sample_factory_examples.tests.test_example_multi
-STATUS_TEST_EXAMPLE_MULTI=$?
+#python -m unittest sample_factory_examples.tests.test_example_multi
+#STATUS_TEST_EXAMPLE_MULTI=$?
 
 echo "Your terminal might be unresponsive (caused by Doom envs manipulating the console). Type reset in your terminal and press Enter to get it back to normal"
-echo "Unit tests status (0 = Success): $STATUS_ALL_TESTS"
+#echo "Unit tests status (0 = Success): $STATUS_ALL_TESTS"
 echo "Custom env status (0 = Success): $STATUS_TEST_EXAMPLE"
```

Comment:
Ok so we're running only one test now. This is cool! Let's fix and add more tests in the coming days! You can use `@unittest.skipUnless` and similar functions to skip individual tests.

Reply:
Yeah, that is where I am stuck, trying to fix other tests😂. Forgive me for my slowness, the code is a little hard for me. I will absolutely learn from Andrew.